overlord/booted.go

```diff
@@ -57,22 +60,56 @@ func SyncBoot() error {
 	kernelSnap, _ := bootloader.GetBootVar("snappy_kernel")
```

Comment:
Why are errors being ignored here? Do we not care if those are completely bogus for whatever reason?